src/fortanix/Logo/Logo.stories.tsx

```diff
@@ -19,18 +19,22 @@ export default {
   },
   tags: ['autodocs'],
   argTypes: {
+    subtitle: { control: 'text' },
+    subtitleTrademark: { control: 'boolean' },
```

Comment:
Was there a reason to specify these `argTypes` explicitly? Storybook should pick this up automatically based on the TypeScript prop types.